src/main/java/jaz/Zer.java

```diff
@@ -50,10 +51,33 @@
                Iterator, Runnable, Callable, Function, Collection, List {
   static final long serialVersionUID = 42L;
 
+  private static final Map<String, Set<String>> HOOKS_USING_JAZ_ZER = new HashMap<String,
+      Set<String>>() {
+    {
```

Comment:
This is silently subclassing `HashMap`, which feels overly intrusive. Since we are iterating over the map anyway, we could just make this a list of map entries or, alternatively, have a small function that initializes the map with statements rather than a single expression.